src/assets/v1.3.0/sign_message_lib.json

```diff
@@ -52,6 +52,8 @@
     "1984": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "2001": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "2008": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
+    "2019": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
+    "2020": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
```

Comment:
Seems not be deployed 
```
reusing "MultiSendCallOnly" at 0x40A2aCCbd92BCA938b02010E17A5b8929b49130D
deploying "SignMessageLib"
```
If I request the bytecode to the rpc, I receive that is empty.

```
curl --location --request POST 'https://rpc.publicmint.io:8545' \
--header 'Content-Type: application/json' \
--data-raw '{
    "jsonrpc": "2.0",
    "method": "eth_getCode",
    "params": [
        "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
        "latest"
    ],
    "id": 1}'
```
Result empty:
```
 {
  "jsonrpc" : "2.0",
  "id" : 1,
  "result" : "0x"
}
```

Reply:
Missing contract has been deployed.